packages/concerto-core/package.json

```diff
@@ -64,9 +64,9 @@
     "yargs": "17.7.2"
   },
   "dependencies": {
-    "@accordproject/concerto-cto": "file:../concerto-cto",
+    "@accordproject/concerto-cto": "*",
```

Comment:
Are we sure we want this? In the future when we release v4, and if some want to install `concerto-core@3.19.9`, won't this forcefully install `concerto-util@4.0.0` and `concerto-cto@4.0.0` which themselves could have a breaking change and thus breaking code for `concerto-core@3.19.9`?

Reply:
This is a test.